gravitee-apim-console-webui/src/management/settings/shared-policy-groups/shared-policy-groups-add-edit-dialog/shared-policy-groups-add-edit-dialog.component.html

```diff
@@ -15,20 +15,22 @@
     limitations under the License.
 
 -->
-<span mat-dialog-title>Add Shared Policy Group for {{ apiTypeLabel }} API</span>
+<span mat-dialog-title>{{ isEdit ? 'Edit' : 'Add' }} Shared Policy Group for {{ apiTypeLabel }} API</span>
 
 <form [formGroup]="formGroup">
   <mat-dialog-content class="mat-typography content">
     <p>Shared Policy Group allow you to create policy collections that can be reused and executed within API flows.</p>
 
     <mat-form-field class="form-field">
       <mat-label>Name</mat-label>
-      <input type="text" aria-label="Name" matInput formControlName="name" />
+      <input #nameInput type="text" aria-label="Name" matInput formControlName="name" />
+      <mat-hint align="end">{{ nameInput.value.length }}/512</mat-hint>
     </mat-form-field>
 
     <mat-form-field class="form-field">
       <mat-label>Description</mat-label>
-      <textarea matInput formControlName="description"></textarea>
+      <textarea #descriptionInput matInput formControlName="description"></textarea>
+      <mat-hint align="end">{{ descriptionInput.value.length }}/1024</mat-hint>
```

Comment:
maybe I'm wrong but I can't see a validator limiting the max length ?

Reply:
haha yes I added it in my next PR with a simple `[maxLength]="512"` ; for the name too